packages/component/src/components/tour-modal/TourModal.tsx

```diff
@@ -60,7 +59,7 @@ export const TourModal: FC<TourModalProps> = ({ open, onClose }) => {
                 onClick={() => setStep(1)}
                 data-testid="onboarding-modal-next-button"
               >
-                Next Tip Please !
+                Next
```

Comment:
i18n?

Reply:
The text has not been finalized yet and I do not intend to upload it to the translation platform. However, it is also possible to add this entry.